index.js

```diff
@@ -306,6 +306,27 @@ exports.isValidPrivate = function (privateKey) {
 }
 
 /**
+ * Checks if the public key satisfies the rules of the curve secp256k1
+ * and the requirements of Ethereum.
+ * @method isValidPublic
+ * @param {Buffer} publicKey The two points of an uncompressed key, unless sanitize is enabled
+ * @param {Boolean} [sanitize=false] Accept public keys in other formats
+ * @return {Boolean}
+ */
+exports.isValidPublic = function (publicKey, sanitize) {
+  if (publicKey.length === 64) {
+    // Convert to SEC1 for secp256k1
+    return secp256k1.publicKeyVerify(Buffer.concat([ new Buffer('04', 'hex'), publicKey ]))
```

Comment:
0x07 for odd -- https://github.com/bitcoin/secp256k1/blob/7b549b1abc06fe1c640014603346b85c8bc83e0b/src/eckey_impl.h#L27

You don't pollute context with `_uncompressedPublicKey`, but actually it's not good solution because use global variables.. I think stay with concat will be better:
```js
return secp256k1.publicKeyVerify(Buffer.concat([ new Buffer([0x04]), publicKey ]))
```